packages/create-sitecore-jss/src/templates/nextjs/package.json

```diff
@@ -71,7 +72,7 @@
   "scripts": {
     "jss": "jss",
     "lint": "eslint ./src/**/*.tsx ./src/**/*.ts ./scripts/**/*.ts",
-    "bootstrap": "ts-node --project tsconfig.scripts.json scripts/bootstrap.ts && graphql-let",
+    "bootstrap": "ts-node --project tsconfig.scripts.json scripts/bootstrap.ts && jss graphql-codegen",
```

Comment:
would it be better to use npm graphql-codegen here? jss is just an alias for npm.
Or better yet - is it possible to add graphql init logic into bootstrap script?

Reply:
I left it here because I think it would be easier for the user to disable code generation at build time if needed. Do you think it would be better to be in the script?